docs/Documentation/Applications/chemicalKinetics.md

```diff
@@ -155,6 +155,34 @@ $ make TPL
 $ make
 ``` 
 
+###Chemkin
+
+Chemkin GUI can be accessed though a [FastX virtual desktop](https://nrel.github.io/HPC/Documentation/Viz_Analytics/virtualgl_fastx/). To use Chemkin Reaction Workbench in a FastX session, open a terminal within the FastX desktop and execute the following:
+```
+$ module load ansys/2025R1
```

Comment:
Does this have to be this Ansys version, or could you change to "module load ansys" so that we don't have to remember to update versions later?